src/app/business/common/components/detail/components/metadata.js

```diff
@@ -35,10 +35,13 @@ const baseDd = css`
 
 const metadataDt = css`
     text-transform: initial;
+    font-family: ${monospaceFamily};
+    margin-left: ${spacingSmall};
```

Comment:
You could use padding-left here, it would avoid having to specify a negative left margin the metadataDd class.